op-challenger/fault/responder.go

```diff
@@ -79,18 +89,25 @@ func (r *faultResponder) BuildTx(ctx context.Context, response types.Claim) ([]b
 	}
 }
 
-// CanResolve determines if the resolve function on the fault dispute game contract
-// would succeed. Returns true if the game can be resolved, otherwise false.
-func (r *faultResponder) CanResolve(ctx context.Context) bool {
+// CallResolve determines if the resolve function on the fault dispute game contract
+// would succeed. Returns the game status if the call would succeed, errors otherwise.
+func (r *faultResponder) CallResolve(ctx context.Context) (uint8, error) {
 	txData, err := r.buildResolveData()
 	if err != nil {
-		return false
+		return 0, err
 	}
-	_, err = r.txMgr.Call(ctx, ethereum.CallMsg{
+	res, err := r.txMgr.Call(ctx, ethereum.CallMsg{
 		To:   &r.fdgAddr,
 		Data: txData,
 	}, nil)
-	return err == nil
+	if err != nil {
+		return 0, err
+	}
+	var status uint8
+	if err = r.fdgAbi.UnpackIntoInterface(&status, "resolve", res); err != nil {
```

Comment:
Ok, this is being hit with a cover profile, but where is the test?

Reply:
How would you like me to test it?

Reply:
I'm asking the location of the test code which tests this. I'm open to lots of ways of testing this, but I'm not seeing an obvious test which tests this. It could be that I'm just missing where the test is.